rslib/backend.proto

```diff
@@ -760,37 +764,58 @@ message BuiltinSearchOrder {
   bool reverse = 2;
 }
 
-message FilterToSearchIn {
-  enum NamedFilter {
-    WHOLE_COLLECTION = 0;
-    CURRENT_DECK = 1;
-    ADDED_TODAY = 2;
-    STUDIED_TODAY = 3;
-    AGAIN_TODAY = 4;
-    NEW = 5;
-    LEARN = 6;
-    REVIEW = 7;
-    DUE = 8;
-    SUSPENDED = 9;
-    BURIED = 10;
-    RED_FLAG = 11;
-    ORANGE_FLAG = 12;
-    GREEN_FLAG = 13;
-    BLUE_FLAG = 14;
-    NO_FLAG = 15;
-    ANY_FLAG = 16;
+message SearchTerm {
+  message Dupe {
+    int64 notetype_id = 1;
+    string first_field = 2;
+  }
+  enum Flag {
+    WITHOUT = 0;
+    ANY = 1;
+    RED = 2;
+    ORANGE = 3;
+    GREEN = 4;
+    BLUE = 5;
+  }
+  enum Rating {
+    ANY_ANSWER_BUTTON = 0;
+    ANSWER_BUTTON_1 = 1;
```

Comment:
Would it make sense to use the names Again, Hard, Good and Easy? I find the term answer button a bit misleading (on the backend it's called ease, which isn't better).
It would be consistent with flags and help to decouple the internally used indices from the frontend representation. It would be escpecially helpful if we get customisable buttons at some point and Good isn't necessarily the third button anymore.

Of course, it's shorter, too, and unambiguous within its namespace.

Reply:
Either way works for me - feel free to change it if you'd like. Ease is a holdover from the DB layer and old code; if you want to rename it AnswerButtonKind that would be an improvement.

Reply:
(or RatingKind, or some other suggestion)